src/Microsoft.DotNet.Cli.Utils/AnsiConsole.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
Is there any reason you are not merging your change? Would be easier if it was just in master.